packages/amplify-util-uibuilder/src/commands/utils/createUiBuilderComponentNew.ts

```diff
@@ -0,0 +1,75 @@
+import { StudioTemplateRendererManager, StudioTemplateRendererFactory, StudioComponent } from '@aws-amplify/codegen-ui-new';
+import {
+  AmplifyRenderer,
+  ReactThemeStudioTemplateRenderer,
+  ReactIndexStudioTemplateRenderer,
+  ModuleKind,
+  ScriptTarget,
+  ScriptKind,
+} from '@aws-amplify/codegen-ui-react-new';
+import { getUiBuilderComponentsPath } from './getUiBuilderComponentsPath';
+import { printer } from 'amplify-prompts';
+import { $TSContext } from 'amplify-cli-core';
+const config = {
+  module: ModuleKind.ES2020,
+  target: ScriptTarget.ES2020,
+  script: ScriptKind.JSX,
+  renderTypeDeclarations: true,
+};
+
+export const createUiBuilderComponent = (context: $TSContext, schema: any) => {
+  const uiBuilderComponentsPath = getUiBuilderComponentsPath(context);
+  const rendererFactory = new StudioTemplateRendererFactory((component: any) => new AmplifyRenderer(component, config) as any);
+
+  const outputPathDir = uiBuilderComponentsPath;
+  const outputConfig = {
+    outputPathDir,
+  };
+
+  const rendererManager = new StudioTemplateRendererManager(rendererFactory, outputConfig);
+
+  rendererManager.renderSchemaToTemplate(schema);
+  return schema;
+};
+
+export const createUiBuilderTheme = (context: $TSContext, schema: any) => {
```

Comment:
this function and above function are almost similar .
can we just pass the ReactThemeStudioTemplateRenderer and AmplifyRenderer as input to only a single function